pkg/api/aim/dao/repositories/run.go

```diff
@@ -398,16 +398,22 @@ func (r RunRepository) SearchRuns(
 
 	log.Debugf("Total runs: %d", total)
 
+	experimentsSelect := database.DB.Select(
+		"ID", "Name",
+	).Where(
+		&models.Experiment{NamespaceID: namespaceID},
+	)
+	// Do not include the filter if there are no ExperimentNames
+	if len(req.ExperimentNames) > 0 {
```

Comment:
I don't think we can support this change -- experiment names are required for fetching data as a way to limit scope, otherwise it can really bog the UI down before the user has even done anything.

We could theoretically add a new function for "50 most recent runs" or something to show on the overview page?

Reply:
I've reverted it. I'll have to work through the alternative path of getting the UI to hit a different endpoint. That will be in a sep PR.